reader/views.py

```diff
@@ -3560,6 +3562,43 @@ def profile_follow_api(request, ftype, slug):
     return jsonResponse({"error": "Unsupported HTTP method."})
 
 
+@catch_error_as_json
+def topic_delete_photo(request, file):
```

Comment:
In the future, we should definitely look to implement more restful api endpoints that handle both POST and DELETE under the same API